swagger.json

```diff
@@ -3,7 +3,7 @@
     "info": {
         "version": "1.0.0",
         "title": "Authors Haven",
-        "description": "Authors Haven is Social platform for the creative at heart",
+        "description": "AAuthors Haven is a social platform for the creative at heart. It is a platform where like minded authors can express themselves in writing",
```

Comment:
### This wrote AAuthors, So remove one A

Reply:
Cool! done